### PR TITLE
GOBNS-1384 GOBNS-1385 ld update

### PR DESCRIPTION
**Pull request by michaelhope-ifit**
bringing the LD package up to 3.1.6 and changing dependencies accordingly

### CODE REVIEW

**Review by parkerjifit — APPROVED** [review]
Looks good!

File: LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\..\tmp\nuget\Xamarin.AndroidX.Migration.1.0.8\build\monoandroid90\Xamarin.AndroidX.Migration.props" Condition="Exists('..\..\..\..\..\tmp\nuget\Xamarin.AndroidX.Migration.1.0.8\build\monoandroid90\Xamarin.AndroidX.Migration.props')" />
```
Comment (jacobt-ifit) [inline LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj]: These relative paths should probably be absolute (see the `Import` element below).

File: LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj [hunk 2]
```diff
@@ -65,18 +70,34 @@
...
+    <EmbeddedReferenceJar Include="Jars\okhttp-4.9.2.jar" />
+    <EmbeddedReferenceJar Include="Jars\okhttp-eventsource-1.11.2.jar" />
     <EmbeddedReferenceJar Include="Jars\slf4j-api-1.7.22.jar" />
-    <EmbeddedReferenceJar Include="Jars\gson-2.8.2.jar" />
+    <EmbeddedReferenceJar Include="Jars\timber-5.0.1.aar" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.Bindings.targets" />
   <Import Project="\tmp\nuget\NuGet.Build.Packaging.0.2.2\build\NuGet.Build.Packaging.targets" Condition="Exists('\tmp\nuget\NuGet.Build.Packaging.0.2.2\build\NuGet.Build.Packaging.targets')" />
   <Import Project="\tmp\nuget\Xamarin.Android.Support.Annotations.26.0.2\build\MonoAndroid80\Xamarin.Android.Support.Annotations.targets" Condition="Exists('\tmp\nuget\Xamarin.Android.Support.Annotations.26.0.2\build\MonoAndroid80\Xamarin.Android.Support.Annotations.targets')" />
+  <Import Project="..\..\..\..\..\tmp\nuget\Xamarin.AndroidX.MultiDex.2.0.1.5\build\monoandroid90\Xamarin.AndroidX.MultiDex.targets" Condition="Exists('..\..\..\..\..\tmp\nuget\Xamarin.AndroidX.MultiDex.2.0.1.5\build\monoandroid90\Xamarin.AndroidX.MultiDex.targets')" />
```
Comment (jacobt-ifit) [inline LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj]: More relative paths here.

**Review by jacobt-ifit — CHANGES_REQUESTED** [review]
There are some relative paths in the `.csproj` that should be absolute. Otherwise all looks good!

**Comment by michaelhope-ifit** [comment]
oops, I guess times havent changed much.

**Review by jacobt-ifit — APPROVED** [review]
👍

**Pull request merged**